docs/smart-contracts/tokens/wayfinding.mdx

```diff
@@ -0,0 +1,74 @@
+
+Assets issued on the Stellar network are accessible to smart contracts with the use of that asset’s Stellar Asset Contract (SAC).
+
+### Stellar Asset Contract
+
+The Stellar Asset Contract (SAC) is compiled into the protocol layer and allows smart contracts to interact with assets issued on Stellar. Every Stellar asset has reserved a SAC on the network that can be deployed by anyone who wants to interact with the asset from a contract. The SAC has access to all account (for XLM) and trustline (for all other assets) balances as well as smart contract token balances.
+
+Read more about the SAC [here](./stellar-asset-contract.mdx).
+
+Learn how to deploy a Stellar Asset Contract for an asset in [this How-To Guide](../../smart-contracts/guides/cli/deploy-stellar-asset-contract.mdx).
+
+**Benefits of the SAC:**
+
+- Compatibility: the SAC benefits from Stellar assets' existing interoperability.
+- Cost and resource efficiency: the SAC is built into the protocol-level code that already runs the network. So, there is no need to spin up a Wasm contract’s VM to invoke a function. Each function within the SAC will be more resource-efficient than its custom-coded counterpart.
+- Less work: you don’t have to write an entirely new contract. A Stellar asset’s SAC already exists on the network and just needs to be deployed to be used.
+- Customization: in addition to the customizations for Stellar assets listed above, asset issuers can set a different smart contract as an admin for their asset’s SAC. Making this admin another smart contract means that that address can add a bit of customization and logic to how the asset will authorize balances and trustlines, mint tokens, etc. Admin addresses can be contracts.
+
+**Downside of the SAC:**
+
+- Other than the customization noted above, it is not possible to modify the behavior of Stellar assets or their SAC. If you’re looking to use assets in a way not supported by Stellar assets, you can create your own custom smart contract token using the token interface and all applications that interact with tokens using the token interface will be able to interact with the custom token.
+
+## Custom tokens
+
+If you have a unique use case where the SAC's capabilities are not sufficient, you can create a custom token that implements the [token interface](./token-interface.mdx). The token interface specifies the functions and events a contract must implement to be compatible with applications that use tokens.
+
+The SAC also implements the token interface and applications that interoperate with the token interface can seamlessly interact with Stellar assets and custom tokens.
+
+:::note
+
+Tokens created with smart contracts cannot interact with Stellar assets unless that Stellar asset has a deployed SAC. Anyone can deploy the SAC for a Stellar asset.
+
+:::
```

Comment:
Its not clear what it means to "interact" here. Can we clarify what this interaction would be?

Reply:
Does "transact" work? Or maybe... "Smart contracts cannot use Stellar assets unless that Stellar asset has a deployed SAC." Does that make more sense?

Reply:
"Smart contracts cannot use Stellar assets unless that Stellar asset has a deployed SAC" makes a lot of sense. I think I was confused with "Tokens created with smart contracts" rather than just "smart contracts".

Reply:
Ok awesome, thank you!